homeassistant/components/netgear/device_tracker.py

```diff
@@ -64,7 +64,7 @@ def __init__(self, router: NetgearRouter, device: Dict[str, any]) -> None:
         self._active = True
         self._attrs = {}
 
-    def update(self) -> None:
+    async def async_update(self) -> None:
```

Comment:
This can be a callback, there is no await.

Reply:
And in that case, don't call it `async_update`, because you're overriding a method in that case, name it something custom.